src/unleash.ts

```diff
@@ -45,15 +45,15 @@ export class Unleash extends EventEmitter {
 
   private metrics: Metrics;
 
-  private staticContext: StaticContext;
+  protected staticContext: StaticContext;
 
   private synchronized: boolean = false;
 
   private ready: boolean = false;
 
   private started: boolean = false;
 
-  private metricRegistry = new InMemoryMetricRegistry();
+  protected metricRegistry = new InMemoryMetricRegistry();
```

Comment:
I'd keep the name explicit about impact metrics. Initially I got confused by the similarity between metrics (current exposure metrics) and metricRegistry. Also I'd stick to plural form "metrics" for the sake of consistency with other names